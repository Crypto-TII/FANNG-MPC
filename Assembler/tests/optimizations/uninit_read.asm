addci c0, c1, 2
