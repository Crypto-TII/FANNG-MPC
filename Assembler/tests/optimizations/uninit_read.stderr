error: read from uninitialized register
 --> tests/optimizations/uninit_read.asm:1:1
  |
1 | addci c0, c1, 2
  | ^^^^^^^^^^^^^^^ register 1 was never written to
  |
warning: Write to register is never read
 --> tests/optimizations/uninit_read.asm:1:1
  |
1 | addci c0, c1, 2
  | --------------- register 0 written to but will never be read from again
  |